src/main/build/version.h

```diff
@@ -25,7 +25,7 @@
 #define FC_FIRMWARE_NAME            "EmuFlight"
 #define FC_VERSION_MAJOR            0  // increment when a major release is made (big new feature, etc)
 #define FC_VERSION_MINOR            2  // increment when a minor release is made (small new feature, change etc)
-#define FC_VERSION_PATCH_LEVEL      40 // increment when a bug is fixed
+#define FC_VERSION_PATCH_LEVEL      47 // increment when a bug is fixed
```

Comment:
please start bumping version seperately cause its obstructive to work as a team

Reply:
not entirely sure what you mean by this. I've been slowly bumping this as I have been testing it in order to easily understand what version testers have been using. I can make this 41 though if you want.

Reply:
https://github.com/emuflight/EmuFlight/pull/119 introduced a less single-individual way of doing releases. you'd bump the version as part of a release process and keep the same version as long as the team... ya well.. decides on when to bump to the next version.
you work on one version until it is finished and just not bump each and any time.
without any roadmap it might be easy for you but it's not good for the project.